src/scep/Client/certificate.py

```diff
@@ -84,7 +84,9 @@ def end_date(self):
 
     @property
     def key_usage(self):
-        return self._certificate.key_usage_value.native
+        if self._certificate.key_usage_value is not None:
```

Comment:
Hi @aren , we still need condition since somehow it is fetched the SCEP CA and instantiating the certificate class.

```
DEBUG:urllib3.connectionpool:https://api.foxbox.foxpass.com:443 "GET /scep/bce64d69-53df-413e-ad18-9f4e4947621b/?operation=GetCACert&message= HTTP/1.1" 200 1537
DEBUG:scep.Client.client:Received response with CA certificates
Error 'NoneType' object has no attribute 'native'
Traceback (most recent call last):
```